resources/js/src/app/shared/interfaces.ts

```diff
@@ -32,9 +32,14 @@ export interface Facet {
 }
 
 export interface FacetValue {
+    count?: string;
+    image?: string;
     id?: string;
     name: string;
     imageUrl?: string;
+    items?: [];
+    position?: string;
+    selected?: boolean;
```

Comment:
I would be really good if this wasn't able to be undefined, as a filter is either selected or it isn't and there is no "i do not know" in this case, as a "i do not know" should always be a `false` instead.